Makefile

```diff
@@ -72,15 +72,15 @@ build: windows linux darwin
 
 .PHONY: docker
 docker:
-	docker build . -t $(HUB)/$(PROJECT):$(VERSION)
+	docker build . -t $(HUB)/$(PROJECT):$(VERSION) -t $(HUB)/$(PROJECT):latest
 
 .PHONY: docker-push
 docker-push:
-	docker push $(HUB)/$(PROJECT):$(VERSION)
+	docker buildx create --use --driver docker-container --name skywalking_eyes_main
+	docker buildx build --push --platform linux/amd64,linux/arm64 -t $(HUB)/$(PROJECT):$(VERSION) -t $(HUB)/$(PROJECT):latest . || true
```

Comment:
Can we save the exit code of this command if it failed and fail the whole make recipe after we clean up the builder?

This is important in the CI so we know there is something wrong instead of failed silently

Reply:
How would I save the exit code of only the first line before OR(`|| `) as this is not a pipe command and `$?` will give me the output of whole buildx command including `|| true`.

Reply:
@mohammedtabish0 can you try this style?

```makefile
docker-push:
	@{ \
		echo 'the build command' ; \
		code=$$? ; \
		echo 'clean up' ; \
		exit $$code ; \
	}
```